### ISSUE TITLE
Seaborn api change causing error in Dimensionality.ipynb

### ISSUE THREAD

**Issue by msneubauer** [illinois-mla/phys-398-mla-image#10]
See https://github.com/PrairieLearn/pl-phys398mla/issues/2

**Comment by matthewfeickert** [comment]
Full traceback from

```python
import numpy as np
import pandas as pd
import seaborn as sns
sns.set()

gen = np.random.RandomState(seed=123)
data = pd.DataFrame()
data['x'] = gen.uniform(-1, +1, 50)
data['y'] = np.zeros_like(data['x'])
sns.jointplot('x', 'y', data, stat_func=None);
```

```pytb
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Input In [7], in <module>
      3 data['x'] = gen.uniform(-1, +1, 50)
      4 data['y'] = np.zeros_like(data['x'])
----> 5 sns.jointplot('x', 'y', data, stat_func=None)

File /opt/conda/lib/python3.9/site-packages/seaborn/_decorators.py:46, in _deprecate_positional_args.<locals>.inner_f(*args, **kwargs)
     36     warnings.warn(
     37         "Pass the following variable{} as {}keyword arg{}: {}. "
     38         "From version 0.12, the only valid positional argument "
   (...)
     43         FutureWarning
     44     )
     45 kwargs.update({k: arg for k, arg in zip(sig.parameters, args)})
---> 46 return f(**kwargs)

File /opt/conda/lib/python3.9/site-packages/seaborn/axisgrid.py:2244, in jointplot(x, y, data, kind, color, height, ratio, space, dropna, xlim, ylim, marginal_ticks, joint_kws, marginal_kws, hue, palette, hue_order, hue_norm, **kwargs)
   2241 if kind.startswith("scatter"):
   2243     joint_kws.setdefault("color", color)
-> 2244     grid.plot_joint(scatterplot, **joint_kws)
   2246     if grid.hue is None:
   2247         marg_func = histplot

File /opt/conda/lib/python3.9/site-packages/seaborn/axisgrid.py:1806, in JointGrid.plot_joint(self, func, **kwargs)
   1803     self._inject_kwargs(func, kwargs, self._hue_params)
   1805 if str(func.__module__).startswith("seaborn"):
-> 1806     func(x=self.x, y=self.y, **kwargs)
   1807 else:
   1808     func(self.x, self.y, **kwargs)

File /opt/conda/lib/python3.9/site-packages/seaborn/_decorators.py:46, in _deprecate_positional_args.<locals>.inner_f(*args, **kwargs)
     36     warnings.warn(
     37         "Pass the following variable{} as {}keyword arg{}: {}. "
     38         "From version 0.12, the only valid positional argument "
   (...)
     43         FutureWarning
     44     )
     45 kwargs.update({k: arg for k, arg in zip(sig.parameters, args)})
---> 46 return f(**kwargs)

File /opt/conda/lib/python3.9/site-packages/seaborn/relational.py:827, in scatterplot(x, y, hue, style, size, data, palette, hue_order, hue_norm, sizes, size_order, size_norm, markers, style_order, x_bins, y_bins, units, estimator, ci, n_boot, alpha, x_jitter, y_jitter, legend, ax, **kwargs)
    823     return ax
    825 p._attach(ax)
--> 827 p.plot(ax, kwargs)
    829 return ax

File /opt/conda/lib/python3.9/site-packages/seaborn/relational.py:608, in _ScatterPlotter.plot(self, ax, kws)
    603 scout_size = max(
    604     np.atleast_1d(kws.get("s", [])).shape[0],
    605     np.atleast_1d(kws.get("c", [])).shape[0],
    606 )
    607 scout_x = scout_y = np.full(scout_size, np.nan)
--> 608 scout = ax.scatter(scout_x, scout_y, **kws)
    609 s = kws.pop("s", scout.get_sizes())
    610 c = kws.pop("c", scout.get_facecolors())

File /opt/conda/lib/python3.9/site-packages/matplotlib/__init__.py:1412, in _preprocess_data.<locals>.inner(ax, data, *args, **kwargs)
   1409 @functools.wraps(func)
   1410 def inner(ax, *args, data=None, **kwargs):
   1411     if data is None:
-> 1412         return func(ax, *map(sanitize_sequence, args), **kwargs)
   1414     bound = new_sig.bind(ax, *args, **kwargs)
   1415     auto_label = (bound.arguments.get(label_namer)
   1416                   or bound.kwargs.get(label_namer))

File /opt/conda/lib/python3.9/site-packages/matplotlib/axes/_axes.py:4468, in Axes.scatter(self, x, y, s, c, marker, cmap, norm, vmin, vmax, alpha, linewidths, edgecolors, plotnonfinite, **kwargs)
   4458 collection = mcoll.PathCollection(
   4459         (path,), scales,
   4460         facecolors=colors,
   (...)
   4465         alpha=alpha
   4466         )
   4467 collection.set_transform(mtransforms.IdentityTransform())
-> 4468 collection.update(kwargs)
   4470 if colors is None:
   4471     collection.set_array(c)

File /opt/conda/lib/python3.9/site-packages/matplotlib/artist.py:1064, in Artist.update(self, props)
   1062             func = getattr(self, f"set_{k}", None)
   1063             if not callable(func):
-> 1064                 raise AttributeError(f"{type(self).__name__!r} object "
   1065                                      f"has no property {k!r}")
   1066             ret.append(func(v))
   1067 if ret:

AttributeError: 'PathCollection' object has no property 'stat_func'
```

**Comment by matthewfeickert** [comment]
Quick fix: Downgrade `seaborn` to `v0.10.1`.
